pages/about.md

```diff
@@ -0,0 +1,40 @@
+<script>{
+	"title": "About QUnit",
+	"pageTemplate": "page-contentfull.php"
+}</script>
+
+QUnit is a powerful, easy-to-use JavaScript unit testing framework. It's used by the jQuery, jQuery UI, jQuery Mobile and many other projects and is capable of testing any generic JavaScript code, including itself!
+
+## Vision and Goals
+
+### Simple setup
+
+Setting up the first unit test with QUnit should be super simple, requiring as little setup as possible.
+
+### Immediate, detailed feedback
+
+When a test or assertion fails, QUnit should provide feedback to the developer as fast as possible, with enough detail to quickly figure out the underlying issue.
+
+### (What else?)
```

Comment:
- performance. QUnit is not only easy, but fast.
- opinionated and lean API, but extensible.
- compatible. It works on many different environments.